cpu/native/startup.c

```diff
@@ -320,6 +333,11 @@ void usage_exit(int status)
 "        The ZEP interface connects to the remote address and may listen\n"
 "        on a local address.\n"
 "        Required to be provided SOCKET_ZEP_MAX times\n"
+#endif
+#ifdef MODULE_NATIVE_CLI_EUI_PROVIDER
+"    -Z <eui64>, --eui64=<eui64>\n"
```

Comment:
I know where this is coming from, but wouldn't e.g. `-U` be a much more sensible short-hand for eUi64 (given `E` is already used)?

Reply:
Ok